AppiumForMac/Server/Handlers/AfMHandlers.m

```diff
@@ -963,6 +963,28 @@ - (AppiumMacHTTPJSONResponse *)post_doubleclick:(NSString*)path data:(NSData *)p
     }];
 }
 
+
+// POST /session/:sessionId/element/:id/scrollTo
+// Scroll to the particular element passed to Appium. Scrolls till the bottom of the screen till the element becomes visible
```

Comment:
is it always the case we need to scroll to the bottom? what if the destination element is located in the other direction, for example upper or at the right side?